worksheet_B/README.md

```diff
@@ -0,0 +1,60 @@
+INPUT random 5 letter word from given list i.e."Sheep", "Towel", "Light", "Learn", "Third", "Every"
+
+DECLARE word variable
+
+DECLARE correctanswer variable
+
+DECLARE guesscount variable
+
+SET correctanswer to False
+
+SET guesscount to 0
+
+WHILE guesscount is less than 4
+
+    DISPLAY 'You have a 5 letter word'
+
+    DISPLAY 'Can you guess what it is? Enter here:'
+
+	  User INPUT guess
+
+	  DECLARE guess variable
+
+	  IF guess is equal to 5
+		    ARRAY word variable, DECLARE as 'wordlist'
+		    ARRAY guess variable, DECLARE as 'guesslist'
+		    SET matching to 0
+
+        FOR i in range(0, 5):
+		  IF guesslist [i] is the same as wordlist [i]:
+			SET matching to matching + 1
+		IF matching is 0
+			DISPLAY 'Score: 0/5'
+			SET guesscount to guesscount + 1
+		IF matching is 1
+			DISPLAY 'Score: 1/5'
+			SET guesscount to guesscount + 1
+		IF matching is 2
+			DISPLAY 'Score: 2/5'
+			SET guesscount to guesscount + 1
+		IF matching is 3
+			DISPLAY 'Score: 3/5'
+			SET guesscount to guesscount + 1
+		IF matching is 4
+			DISPLAY 'Score: 4/5'
+			SET guesscount to guesscount + 1
+		IF matching is 5
+			DISPLAY 'Correct!'
+			SET correctanswer to True
+			SET guesscount to 4
+
+    ELSE, DISPLAY 'You need to guess a FIVE letter word'
+	END WHILE
+
+IF correctanswer is False
+	DISPLAY '4 incorrect entries, Game Over"
+
+ELSE
+	DISPLAY 'Well done!
+	
+https://cloud.githubusercontent.com/assets/22060783/19214237/694f8968-8d76-11e6-998b-c2ede6d8c387.png	
```

Comment:
Some arrows appear to be missing from your flowchart, which makes it difficult to follow -- every node except the start node should have at least one arrow coming into it, and every node except the end node should have at least one arrow coming out of it.

It looks like you've got one node in your flowchart for every line of your pseudocode, which isn't necessarily the best approach. For example it's not really necessary to declare variables as a separate process node. Also loops in a flowchart are generally implied by the flow (i.e. by a path looping back on itself), and don't need to be mentioned explicitly.